lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+require 'twilio-ruby'
+# require_relative '../.env.rb'
+
+class Takeaway
+
+  def menu
+    @menu = { 'Tomato soup' => 5,
+              'Steak and chips' => 15,
+              'Salmon Stew' => 12,
+              'Chicken currey' => 10,
+              'Beef casserole' => 18 }
+  end
```

Comment:
This method is very confusing. You are asking for a menu, which sets an instance variable equal to a hash, and returns that hash.

Until you call this method, an instance of the Takeaway class has no `@menu` at all.

You should move the instance variable assignation to an `initialize` method, and then make the `menu` method _return_ the instance variable `@menu` you create. The last bit is basically what an `attr_reader` does.